versioned_docs/version-v1.10.0/sdk-java/01-intro.md

```diff
@@ -30,12 +30,10 @@ To install EventMesh SDK for Java with Maven, declare `org.apache.eventmesh:even
   <dependency>
     <groupId>org.apache.eventmesh</groupId>
     <artifactId>eventmesh-sdk-java</artifactId>
-    <version>1.4.0</version>
+    <version>1.9.0</version>
```

Comment:
This should be `1.10.0`.

Reply:
Fixed in https://github.com/apache/eventmesh-site/pull/170/commits/dad1f5807f4146a7b06f8eb45e8cb350f5e7f6ae